### DISCUSSION TITLE
Allow to save searches

### DISCUSSION THREAD

**Discussion by yagueto** [community/community#6123] [Other Feature Feedback, Questions, & Ideas]
It would be useful for me if GitHub allowed me to save searches. For example: now that we are in hacktoberfest time, I find myself regularly repeating in the search box `is:issue is:open label:hacktoberfest …` for searching for issues to contribute out there.

**Comment by mrienstra** [comment]
Obviously you can just use a browser bookmark, e.g. https://github.com/search?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest , but I'm guessing you want this to be saved in your GitHub account somewhere -- where do you imagine you'd find this? Not clear that https://github.com/notifications?query=is%3Asaved would make sense, maybe https://github.com/search ?

**Reply by yagueto** [reply]
Yeah, using a bookmark seems reasonable, but since I use multiple devices saving it in my account would be easier.

I think that finding it in the dashboard and in the explore section would be rather intuitive
